### PR TITLE
Feature/analyses final ee2

### PR DESCRIPTION
**Pull request by PerryShafran-NOAA**
## Pull Request Testing ##

- [ ] Describe testing already performed for this Pull Request:</br>

EE2 edits:

- cyc is changed to vhr throughout the code.  $cycle is used *only* to run setpdy in the J-job.  $cycle is defined as t${vhr}z.  
- COMINobs in stats job changed to COMINobsproc
- COMINfcst in stats job changed to COMINrtma and COMINurma
- COMINanl in plots job changed to EVSINanl
- The full metplus log file is appended to the stdout.  It was not possible to remove the output to the screen; this should be fixed in later versions of METplus and thus will be fixed for EVSv2.  (The plot log file had been appended in a previous PR.)

- [ ] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

Test the stats and plots jobs.  
When running the stats job used qsub -v vhr=04 or your favorite cycle.
When running the plots job, set EVSINanl to the emc.vpppg stats output.

- [ ] Has the code been checked to ensure that no errors occur during the execution? **[Yes or No]**

- [ ] Do these updates/additions include sufficient testing updates? **[Yes or No]**

- [ ] Please complete this pull request review by **[Fill in date]**.</br>

ASAP.  

## Pull Request Checklist ##

- [ ] Review the source issue metadata (required labels, projects, and milestone).
- [ ] Complete the PR description above.
- [ ] Ensure the PR title matches the feature branch name.
- [ ] Check the following:
- [ ]  Instructions provided on how to run
- [ ]  Developer's name is replaced by ${user} where necessary throughout the code
- [ ]  Check that the ecf file has all the proper definitions of variables
- [ ]  Check that the jobs file has all the proper settings of COMIN and COMOUT and other input variables
- [ ]  Check to see that the output directory structure is followed
- [ ]  Be sure that you are not using MET utilities outside the METplus wrapper structure

- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue.

### CODE REVIEW

File: dev/drivers/scripts/plots/analyses/jevs_analyses_grid2obs_plots.sh [hunk 1]
```diff
@@ -46,11 +46,11 @@ export job=${PBS_JOBNAME:-jevs_${MODELNAME}_${VERIF_CASE}_${STEP}}
 export jobid=$job.${PBS_JOBID:-$$}
 
 
-export COMIN=/lfs/h2/emc/vpppg/noscrub/$USER/${NET}/${evs_ver}
+export COMIN=/lfs/h2/emc/vpppg/noscrub/emc.vpppg/${NET}/${evs_ver}
```
Comment (ShelleyMelchior-NOAA) [inline dev/drivers/scripts/plots/analyses/jevs_analyses_grid2obs_plots.sh]: 😉 you know what to do here

File: dev/drivers/scripts/stats/analyses/jevs_analyses_rtma_grid2obs_stats.sh [hunk 2]
```diff
@@ -34,7 +34,7 @@ module load prod_envir/${prod_envir_ver}
 export envir=prod
 export DATAROOT=/lfs/h2/emc/stmp/${USER}/evs_test/$envir/tmp
 export KEEPDATA=YES
-export SENDMAIL=YES
+export SENDMAIL=NO
```
Comment (ShelleyMelchior-NOAA) [inline dev/drivers/scripts/stats/analyses/jevs_analyses_rtma_grid2obs_stats.sh]: Revert back to YES?

**Comment by ShelleyMelchior-NOAA** [comment]
@PerryShafran-NOAA , curious, how much larger are the log files w/ them all cat'd together? I will run and find out, but curious if you knew off the top of your head. Analyses is a small component. The other larger components might produce some really unwieldy log files.

**Comment by PerryShafran-NOAA** [comment]
Not so large for analyses.  Don't know the exact size.  And yeah, they will become large for other components, but they seem to be necessary for EE2, so that's how it has to be.

File: jobs/JEVS_ANALYSES_STATS [hunk 3]
```diff
@@ -67,8 +67,9 @@ if [ "$modsys" = "rtma" ] ; then
   mod_ver=$rtma_ver
 fi
 export COMIN=${COMIN:-$(compath.py ${envir}/com/${NET}/$evs_ver)}
-export COMINfcst=${COMINfcst:-$(compath.py ${envir}/com/${modsys}/${mod_ver})}
-export COMINobs=${COMINobs:-$(compath.py ${envir}/com/obsproc/${obsproc_ver})}
+export COMINrtma=${COMINrtma:-$(compath.py ${envir}/com/${modsys}/${mod_ver})}
+export COMINurma=${COMINurma:-$(compath.py ${envir}/com/${modsys}/${mod_ver})}
+export COMINobsproc=${COMINobs:-$(compath.py ${envir}/com/obsproc/${obsproc_ver})}
```
Comment (ShelleyMelchior-NOAA) [inline jobs/JEVS_ANALYSES_STATS]: I think line 72 should change to: 
```
export COMINobsproc=${COMINobsproc:-$(compath.py ${envir}/com/obsproc/${obsproc_ver})}
```

Comment (PerryShafran-NOAA) [reply jobs/JEVS_ANALYSES_STATS]: Right you are!  I'll change that.

Comment (PerryShafran-NOAA) [reply jobs/JEVS_ANALYSES_STATS]: Should be fixed.

**Comment by ShelleyMelchior-NOAA** [comment]
The analyses stats have been run for vhr=04.
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/stats/analyses/atmos.20231011/*/grid2obs`
logfiles: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_analyses_*_grid2obs_stats.o1061496*`

**Comment by PerryShafran-NOAA** [comment]
The stats look good.

**Comment by ShelleyMelchior-NOAA** [comment]
> The stats look good.

Are the logfiles as expected, too, w/ the new cat'ing behavior?

**Comment by PerryShafran-NOAA** [comment]
Yes, I see the full log files printed in the output.

**Comment by ShelleyMelchior-NOAA** [comment]
The analyses plots processing has completed.
COMOUT=`/lfs/h2/emc/vpppg/noscrub/shelley.melchior/evs/v1.0/plots/analyses/atmos.20231011`
logfile: `/lfs/h2/emc/ptmp/shelley.melchior/EVS_out/jevs_analyses_grid2obs_plots.o106152555`

**Comment by PerryShafran-NOAA** [comment]
The plot tar file looks good, and I do see the log files being printed.

**Comment by ShelleyMelchior-NOAA** [comment]
Excellent! I think we need a sync fork and I will do the formal code review prior to approval for  merge.

File: jobs/JEVS_ANALYSES_PLOTS [hunk 4]
```diff
@@ -57,7 +57,7 @@ export RUN=${RUN:-atmos}
 echo $COMPATH
 
 export COMIN=${COMIN:-$(compath.py ${envir}/com/${NET}/${evs_ver})}
-export COMINanl=${COMINanl:-${COMIN}/stats/${COMPONENT}}
+export EVSINanl=${EVSINanl:-${COMIN}/stats/${COMPONENT}}
```
Comment (ShelleyMelchior-NOAA) [inline jobs/JEVS_ANALYSES_PLOTS:60]: Just a note here ... EVSINanl was not defined in the driver script. But this worked fine b/c COMIN is defined in the driver script and used to define EVSINanl in the J-job.

Comment (PerryShafran-NOAA) [reply jobs/JEVS_ANALYSES_PLOTS:60]: Oh.  So should EVSINanl be defined in the driver?

**Comment by AliciaBentley-NOAA** [comment]
@PerryShafran-NOAA I've looked through this PR and I think that it looks excellent! My only question is about $COMINrtma. It looks like you are using $COMINrtma in both your "rtma" and "rtma_ru" scripts. Where do you specify that you want the rtma_ru files in your rtma_ru scripts? 

CC @ShelleyMelchior-NOAA

**Comment by ShelleyMelchior-NOAA** [comment]
> @PerryShafran-NOAA I've looked through this PR and I think that it looks excellent! My only question is about $COMINrtma. It looks like you are using $COMINrtma in both your "rtma" and "rtma_ru" scripts. Where do you specify that you want the rtma_ru files in your rtma_ru scripts?
> 
> CC @ShelleyMelchior-NOAA

@AliciaBentley-NOAA , rtma_ru data are in the rtma com space.

File: jobs/JEVS_ANALYSES_PLOTS (hunk 4 above)
Comment (AliciaBentley-NOAA) [reply jobs/JEVS_ANALYSES_PLOTS:60]: @PerryShafran-NOAA I think this is fine as it is! Defining COMIN in the dev drivers was intended to make everything that uses $COMIN in the J-jobs work!

**Comment by AliciaBentley-NOAA** [comment]
@ShelleyMelchior-NOAA Correct. I can't find where we're making a distinction between RTMA and RTMA RU files, though. I wanted to make sure we weren't accidentally doing the RTMA verification again and calling it RTMA RU. I'd like Perry to point me to where this is done:

[rtma2p5.20231012/](https://nomads.ncep.noaa.gov/pub/data/nccf/com/rtma/v2.10/rtma2p5.20231012/) 
[rtma2p5_ru.20231012/](https://nomads.ncep.noaa.gov/pub/data/nccf/com/rtma/v2.10/rtma2p5_ru.20231012/)

**Comment by PerryShafran-NOAA** [comment]
In the exevs scripts file, I set modnam to rtma2p5_ru, which is the directory name under rtma that the rtma_ru files reside.  This variable is used to find the rtma2p5_ru directory and files underneath.  

The .o file would demonstrate that it's the correct file, and I would have caught that long ago if it were wrong.

**Comment by ShelleyMelchior-NOAA** [comment]
> In the exevs scripts file, I set modnam to rtma2p5_ru, which is the directory name under rtma that the rtma_ru files reside. This variable is used to find the rtma2p5_ru directory and files underneath.
> 
> The .o file would demonstrate that it's the correct file, and I would have caught that long ago if it were wrong.

Yep, $modnam in the exevs scripts.

**Comment by AliciaBentley-NOAA** [comment]
@PerryShafran-NOAA Thanks. I just wanted to know where it was done to understand your Regional code better.

File: jobs/JEVS_ANALYSES_PLOTS (hunk 4 above)
Comment (ShelleyMelchior-NOAA) [reply jobs/JEVS_ANALYSES_PLOTS:60]: I agree -- I think this is fine. I was pointing it out b/c the PR instructions indicated to change EVSINanl, but I had changed COMIN in the driver instead.

**Review by ShelleyMelchior-NOAA — APPROVED** [review]
Testing yields expected results. This PR is approved for merge.

**Pull request merged**